docs/lib/datastore/fragments/ios/getting-started/50_initDataStore.md

```diff
@@ -14,6 +21,12 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
         print("Failed to initialize Amplify with \(error)")
         return false
     }
+
     return true
 }
 ```
+Upon building and running this application you should see the following in your console window:
+
+```console
+Amplify configured with DataStore plugin
```

Comment:
Similar to Android comment, is there a Logger on Amplify core to enable debug messages?